vsintegration/src/FSharp.Editor/Commands/FsiCommandService.fs

```diff
@@ -36,15 +36,14 @@ type internal FsiCommandFilter(serviceProvider: System.IServiceProvider) =
 
     interface IOleCommandTarget with
         member x.Exec (pguidCmdGroup, nCmdId, nCmdexecopt, pvaIn, pvaOut) =
-            if pguidCmdGroup = VSConstants.VsStd11 then
-                if nCmdId = uint32 VSConstants.VSStd11CmdID.ExecuteSelectionInInteractive then
-                    Hooks.OnMLSend projectSystemPackage.Value FsiEditorSendAction.ExecuteSelection null null
-                elif nCmdId = uint32 VSConstants.VSStd11CmdID.ExecuteLineInInteractive then
-                    Hooks.OnMLSend projectSystemPackage.Value FsiEditorSendAction.ExecuteLine null null
+            if pguidCmdGroup = VSConstants.VsStd11 && nCmdId = uint32 VSConstants.VSStd11CmdID.ExecuteSelectionInInteractive then
+                do Hooks.OnMLSend projectSystemPackage.Value FsiEditorSendAction.ExecuteSelection null null
```

Comment:
I'm brand new to F# so I'm willing to respect whatever coding conventions are in place for this repo. I've go no strong feelings one way or the other so I can remove them if people prefer.